### PR TITLE
[TD]DPG spacing for oblique views

### PR DESCRIPTION
**Pull request by WandererFan**
Thank you for creating a pull request to contribute to FreeCAD! To ease integration, please confirm the following:

- [ ] Branch rebased on latest master `git pull --rebase upstream master`
- [ ] Unit tests confirmed to pass by running `./bin/FreeCAD --run-test 0`
- [ ] Commit message is [well-written](https://chris.beams.io/posts/git-commit/)
- [ ] Commit message includes `issue #<id>` or `fixes #<id>` where `<id>` is the [associated MantisBT](https://freecadweb.org/wiki/tracker#GitHub_and_MantisBT) issue id if one exists

And please remember to update the Wiki with the features added or changed once this PR is merged.  
**Note**: If you don't have wiki access, then please mention your contribution on the [0.19 Changelog Forum Thread](https://forum.freecadweb.org/viewtopic.php?f=10&t=34586).

---

### CODE REVIEW

File: src/Mod/TechDraw/App/DrawProjGroup.cpp [hunk 1]
```diff
@@ -918,6 +959,7 @@ void DrawProjGroup::makeViewBbs(DrawProjGroupItem *viewPtrs[10],
         bboxes[i] = empty;
         if (viewPtrs[i]) {
             bboxes[i] = viewPtrs[i]->getBoundingBox();
+//            bboxes[i] = viewPtrs[i]->getBoundingBox(viewPtrs[i]->getProjectionCS(Base::Vector3d(0.0, 0.0, 0.0)));
```
Comment (maidenone) [inline src/Mod/TechDraw/App/DrawProjGroup.cpp:962]: remove this line? or motivate why it shall be here

File: src/Mod/TechDraw/Gui/QGIViewPart.cpp [hunk 2]
```diff
@@ -59,6 +59,7 @@
 #include <Mod/TechDraw/App/DrawGeomHatch.h>
 #include <Mod/TechDraw/App/DrawViewDetail.h>
 #include <Mod/TechDraw/App/DrawProjGroupItem.h>
+#include <Mod/TechDraw/App/DrawProjGroup.h>
 #include <Mod/TechDraw/App/Geometry.h>
 #include <Mod/TechDraw/App/Cosmetic.h>
 //#include <Mod/TechDraw/App/Preferences.h>
```
Comment (maidenone) [inline src/Mod/TechDraw/Gui/QGIViewPart.cpp:65]: remove include if not needed

File: src/Mod/TechDraw/App/GeometryObject.cpp [hunk 3]
```diff
@@ -788,6 +788,7 @@ Base::BoundBox3d GeometryObject::calcBoundingBox() const
              BRepBndLib::Add((*it)->occEdge, testBox);
         }
     }
+
```
Comment (maidenone) [inline src/Mod/TechDraw/App/GeometryObject.cpp:791]: accidental spacing?

**Comment by WandererFan** [comment]
> is there a forum thread yet? how do i best test your code?

[forum thread is here](https://www.forum.freecadweb.org/viewtopic.php?f=35&t=47600&sid=223142f3c8dd0f74f6d2d60b3949dbc5
)

**Pull request merged**